app/models/taxon.rb

```diff
@@ -30,4 +30,8 @@ def base_path
   def link_type
     'taxons'
   end
+
+  def internal_name
```

Comment:
Could you please add a test case to these model about this behaviour? Just in case it gets changed in the future.